packages/mdc-snackbar/index.js

```diff
@@ -47,6 +47,16 @@ export class MDCSnackbar extends MDCComponent {
       unsetActionAriaHidden: () => getActionButton().removeAttribute('aria-hidden'),
       setActionText: (text) => { getActionButton().textContent = text; },
       setMessageText: (text) => { getText().textContent = text; },
+      setFocus: () => getActionButton().focus(),
+      visibilityIsHidden: () => document.hidden,
+      registerBlurHandler: (handler) => getActionButton().addEventListener('blur', handler, true),
```

Comment:
Document it definitely helps, but IMHO give hint in naming also helps. Imagine if I am the person implementing those adapters in another framework (instead of using it directly provided by our vanilla adapter), I probably won't know I need to have a capturing eventListener just by looking at this name and confused why the component is not behavior correctly.